components/checkbox/index.css

```diff
@@ -188,16 +358,15 @@
 			cursor: default;
 		}
 
-		/* Overrides disabled styles */
-		.spectrum-Checkbox-input,
-		.spectrum-Checkbox-input:checked {
-			&:disabled + .spectrum-Checkbox-box::before {
-				border-color: var(--highcontrast-checkbox-color-default, var(--mod-checkbox-control-selected-color-default, var(--spectrum-checkbox-control-selected-color-default)));
-				background-color: var(--spectrum-checkbox-checkmark-color);
+		&:hover,
+		.is-focus-visible {
```

Comment:
```suggestion
		&:focus-visible {
```
We shouldn't be using the `.is-focus-visible` class in our CSS, as this class is only added within our Storybook by the build process so we can test pseudo classes on the VRTs. That's done by the `postcss-pseudo-classes` plugin.

This is leading to an incorrect color appearing on the Docs page when I tab focus on the emphasized indeterminate checkbox, which is different than what appears on the testing preview grid (that looks correct):

![Screenshot 2025-04-03 at 2 20 19 PM](https://github.com/user-attachments/assets/6b34eec5-7c04-4dfa-94ba-8416aa835dd0)